src/widgets/panel/utilitypanel.cpp

```diff
@@ -72,7 +73,7 @@ UtilityPanel::clearToolWidget()
 void
 UtilityPanel::pushWidget(QWidget* w)
 {
-  m_layout->addWidget(w);
+  m_layout->insertWidget(m_layout->count() - 1, w);
```

Comment:
If this is being inserted at the end why not use 
```
addWidget()
```

Reply:
It needs to insert widgets before the Close button now, not at the end.